commit 4139fd587320da311fca9c4da231dc850d4f4a22
Author: Thara Gopinath <thara.gopinath@linaro.org>
Date:   Thu Feb 11 15:01:28 2021 -0500

    crypto: qce - Remove totallen and offset in qce_start
    
    totallen is used to get the size of the data to be transformed.
    This is also available via nbytes or cryptlen in the qce_sha_reqctx
    and qce_cipher_ctx. Similarly offset convey nothing for the supported
    encryption and authentication transformations and is always 0.
    Remove these two redundant parameters in qce_start.
    
    Reviewed-by: Bjorn Andersson <bjorn.andersson@linaro.org>
    Signed-off-by: Thara Gopinath <thara.gopinath@linaro.org>
    Signed-off-by: Herbert Xu <herbert@gondor.apana.org.au>

diff --git a/drivers/crypto/qce/sha.c b/drivers/crypto/qce/sha.c
index 2813c9a27a6e..8e6fcf2c21cc 100644
--- a/drivers/crypto/qce/sha.c
+++ b/drivers/crypto/qce/sha.c
@@ -113,7 +113,7 @@ static int qce_ahash_async_req_handle(struct crypto_async_request *async_req)
 
 	qce_dma_issue_pending(&qce->dma);
 
-	ret = qce_start(async_req, tmpl->crypto_alg_type, 0, 0);
+	ret = qce_start(async_req, tmpl->crypto_alg_type);
 	if (ret)
 		goto error_terminate;